### PR TITLE
Hgt detection

### PR DESCRIPTION
**Pull request by kescobo**
`create_blast_db.py` now has function to create temporary fasta file from mongoDB and create a blast database.

Also removed import function that creates list of all records, but retained mongo import of list of records for later use.

### CODE REVIEW

File: src/DataImport/mongo_import.py [hunk 1]
```diff
@@ -5,12 +5,21 @@
...
 
     print db.collection_names(False)
+
+def mongo_import_record(record, collection):
+    """
+    Insert a single `record` into `collection`
+    :param record: gene record
+    :param collection: MongoDB collection name
+    """
+    db[collection].insert_one(record)
```
Comment (lydiaauch) [inline src/DataImport/mongo_import.py]: Add newline at end of file

Comment (kescobo) [inline src/DataImport/mongo_import.py]: Done.

**Comment by kescobo** [comment]
@lydiaauch If you have no objection, I think I want to close this, squash a bunch of the commits and then open a new one - this is super messy,

**Comment by kescobo** [comment]
Closing to clean up the mess I made.

**Pull request closed without merging**